static/js/components/email/EmailCompositionDialog.js

```diff
@@ -38,12 +37,18 @@ const EmailCompositionDialog = (props: EmailDialogProps) => {
     updateEmailEdit,
     open,
     email: { inputs, validationErrors, fetchStatus },
-    searchkit,
     sendEmail,
+    title,
+    children
   } = props;
 
+  let subHeading;
+  if (children) {
+    subHeading = <h5 className="sub-heading">{ children }</h5>;
+  }
```

Comment:
this is actually being changed in the PR for #2284. do you mind reviewing the change as part of that?